src/Build/Logging/SimpleErrorLogger.cs

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.Logging.SimpleErrorLogger
+{
+    /// <summary>
+    /// This logger ignores all message-level output, writing errors and warnings to
+    /// standard error, colored red and yellow respectively.
+    ///
+    /// It is currently used only when the user requests information about specific
+    /// properties, items, or target results. In that case, we write the desired output
+    /// to standard out, but we do not want it polluted with any other kinds of information.
+    /// Users still might want diagnostic information if something goes wrong, so still
+    /// output that as necessary.
+    /// </summary>
+    public class SimpleErrorLogger : INodeLogger
+    {
+        public bool hasLoggedErrors = false;
```

Comment:
Readonly property

Reply:
I write to hasLoggedErrors in HandleErrorEvent. Should this have been on a different line? I can make the other two readonly.

Reply:
Oh, that makes more sense. Coincidentally, ladipro made a similar comment in my other PR. I should be able to pull in that change to cover this 🙂